html_css/web_accessibility.md

```diff
@@ -0,0 +1,70 @@
+### Introduction
+
+At this point in the curriculum you've learned incredibly valuable concepts in whichever full stack path you chose, you've created some awesome, resume worthy projects, and you should have a better understanding of some HTML and CSS concepts. One could even argue that you may now love working with CSS...?
+
+What you may not have too much an understanding of, though, is the topic of accessibility, often referred to as "a11y" (due to there being 11 letters between the first and last letters). Unfortunately, this is a topic that many people either don't know much about, or just don't take into account when developing websites. If you fit into either of those two categories, you may have adopted some habits that aren't exactly a11y friendly. Before we dive into how you can break away from such habits and make more a11y friendly websites, it's important to first understand some basic information when it comes to web accessibility.
+
+### Learning Outcomes
+By the end of this lesson, you should be able to:
+
+* Explain what web accessibility is.
+* Explain the purpose of the Web Content Accessibility Guidelines.
+* Name the 4 principles of the WCAG.
+* Know the differences between the WCAG conformity levels.
+
+### What is Web Accessibility?
+
+Web accessibility is when websites don't have barriers that prevent interaction, access, or contribution from users with a disability, situational limitation, or changing abilities due to age.
+
+There are different types of disabilities, including (but not limited to) auditory, physical/motor, cognitive, or visual. A disability can be permanent, such as a user who is completely blind, or it can be temporary, such as a user with a broken arm, and users may have multiple disabilities. Older users with changing abilities may even have similar conditions as those who have a disability.
```

Comment:
```suggestion
There are different types of disabilities, including (but not limited to) auditory, physical/motor, cognitive, or visual. A disability can be permanent, such as a user who is completely blind, or it can be temporary, such as a user with a broken arm. Users may have multiple disabilities at any given time. Older users with changing abilities may even have similar conditions as those who have a disability.
```

I broke the 'and users may have multiple disabilities' out into its own statement and added a few words for completeness.